Utils/Dataflow/009_oracleConnector/dbConnection.py

```diff
@@ -151,7 +152,10 @@ def execute_saved(self, qname, **params):
             return False
 
         try:
+            t1 = time.time()
             c.execute(None, params)
+            t2 = time.time()
+            sys.stderr.write("(TIMING) (%s) (sec): %s\n" % (qname, t2 - t1))
```

Comment:
@mgolosova 

Why we should output info about execution time (if there is no error) to stderr?

Reply:
@mgolosova 
we have just two ways to output something we need: stderr and stdout. If we send a message to stdout (just an "in-mind" experiment) -- and even create a pipeline --, a message will be among all sended data, so all system logs should be sended to stderr.
if I understood it right,